angular/src/app/player.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+
+import { Player } from './player';
+import { PLAYERS } from './mock-players';
+
+@Injectable()
+export class PlayerService {
```

Comment:
Consider calling this `PongService` and putting _all_ the API calls here. Seems like overkill to make this really granular. You can still abstract away talking to a real server or in-memory test server or whatever, but I can't see why you'd want to talk to different servers for different _parts_ of the API for this app.